Formula/cpr.rb

```diff
@@ -16,23 +16,28 @@ class Cpr < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "openssl@1.1"
 
   uses_from_macos "curl"
 
+  on_linux do
+    depends_on "openssl@3"
+  end
+
   def install
-    args = std_cmake_args + %w[
+    args = std_cmake_args + %W[
       -DCPR_FORCE_USE_SYSTEM_CURL=ON
       -DCPR_BUILD_TESTS=OFF
+      -DCMAKE_INSTALL_RPATH=#{rpath}
     ]
 
-    system "cmake", ".", *args, "-DBUILD_SHARED_LIBS=ON"
-    system "make", "install"
+    system "cmake", "-S", ".", "-B", "build", *args, "-DBUILD_SHARED_LIBS=ON"
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
 
-    system "make", "clean"
-    system "cmake", ".", *args, "-DBUILD_SHARED_LIBS=OFF"
-    system "make"
-    lib.install "lib/libcpr.a"
+    rm_rf "build"
+    system "cmake", "-S", ".", "-B", "build", *args, "-DBUILD_SHARED_LIBS=OFF"
+    system "cmake", "--build", "build"
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build-static", "-DBUILD_SHARED_LIBS=OFF" *args
    system "cmake", "--build", "build-static"
```
Let's not destroy the logs before `brew` has a chance to store them.